src/transport/tls/listen.rs

```diff
@@ -173,8 +173,7 @@ impl<L: HasConfig, G> Listen<L, G> {
             });
 
             incoming
-                .take(connection_limit)
```

Comment:
I believe that at one point, the ability to limit the number of accepted connections over the lifetime of the process was required for some kind of testing thing? I can no longer remember the rationale, but is it still necessary?

Reply:
Given that we run this with a MAX value, no.